config/currency_iso.yml

```diff
@@ -2475,6 +2475,21 @@ xcd:
   thousands_separator: ","
   iso_numeric: '951'
   smallest_denomination: 1
+xcg:
+  priority: 100
+  iso_code: XCG
+  name: Caribbean Guilder
+  symbol: Cg
```

Comment:
Just calling this out, the symbol wasn't correct here and i've corrected it before merging